### PR TITLE
chore(docs): add how-to's for angular material inclusion

### PR DESCRIPTION
**Pull request by jeremymwells**
applies to [#2711](https://github.com/angular/angular-cli/issues/2711)

### CODE REVIEW

File: docs/documentation/stories/include-angular-material.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+```
+ng new my-app && cd my-app
+```
+
+With the new project created and ready, you will next need to install Angular Material.
+
+Install the `@angular/material` library and add the dependency to package.json...
+```bash
+npm install --save @angular/material
+
```
Comment (hansl) [inline docs/documentation/stories/include-angular-material.md]: Remove extra line

File: docs/documentation/stories/include-angular-material.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
+# Include [Angular Material](https://material.angular.io)
+
+[Angular Material](https://material.angular.io) is a set of Material Design components for Angular apps. 
+This guide will walk you through adding material design to your Angular CLI project and configuring it to use Angular Material.
+
+Create a new project and navigate into the project...
+```
+ng new my-app && cd my-app
```
Comment (hansl) [inline docs/documentation/stories/include-angular-material.md]: Split into two lines.

File: docs/documentation/stories/include-angular-material.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+  ],
+  ...
+})
+
+```
+
+Now that the project is set up, it must be configured to include the CSS for a theme. Angular Material ships with some prebuilt theming, which is located in `node_modules/@angular/material/core/theming/prebuilt`.
+
+To add an angular CSS theme and material icons to your app...
+```css
```
Comment (hansl) [inline docs/documentation/stories/include-angular-material.md]: This is `sass` typing, not `css`.

File: docs/documentation/stories/include-angular-material.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+After saving this file, return to the browser to see the angular material styled button.
+
+### More Info 
+
+ - [Getting Started](https://material.angular.io/guide/getting-started)
+ - [Theming Angular Material](https://material.angular.io/guide/theming)
+ - [Theming your own components](https://material.angular.io/guide/theming-your-components)
+
+# Include [Flex Layout](https://github.com/angular/flex-layout) for [Angular Material](https://material.angular.io)
```
Comment (hansl) [inline docs/documentation/stories/include-angular-material.md]: I think `Flex Layout` is a separate library and should have its own guide, since you can use it without Material. I'm asking around.

File: docs/documentation/stories/include-angular-material.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+
+[Angular Material](https://material.angular.io) is a set of Material Design components for Angular apps. 
+This guide will walk you through adding material design to your Angular CLI project and configuring it to use Angular Material.
+
+Create a new project and navigate into the project...
+```
+ng new my-app && cd my-app
+```
+
+With the new project created and ready, you will next need to install Angular Material.
```
Comment (hansl) [inline docs/documentation/stories/include-angular-material.md]: Remove this line, it's just repeating the lines above and below it.

File: docs/documentation/stories/include-angular-material.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+
+Install the `@angular/material` library and add the dependency to package.json...
+```bash
+npm install --save @angular/material
+
+```
+
+Import the Angular Material NgModule into your app module...
+```javascript
+//in src/app/app.module.ts -->
```
Comment (hansl) [inline docs/documentation/stories/include-angular-material.md]: Remove `-->`?

File: docs/documentation/stories/include-angular-material.md [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+// other imports 
+
+@NgModule({
+  imports: [
+    ...
+    MaterialModule.forRoot()
+  ],
+  ...
+})
+
```
Comment (hansl) [inline docs/documentation/stories/include-angular-material.md]: Remove extra empty line.

File: docs/documentation/stories/include-angular-material.md [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+```
+
+Now that the project is set up, it must be configured to include the CSS for a theme. Angular Material ships with some prebuilt theming, which is located in `node_modules/@angular/material/core/theming/prebuilt`.
```
Comment (hansl) [inline docs/documentation/stories/include-angular-material.md]: `prebuilt/` is a directory, you should end that path with an extra `/` to indicate so.

File: docs/documentation/stories/include-angular-material.md [hunk 9]
```diff
@@ -0,0 +1,152 @@
...
+
+To add an angular CSS theme and material icons to your app...
+```css
+/* in src/styles.css */
+
+@import '~@angular/material/core/theming/prebuilt/deeppurple-amber.css';
+@import '~https://fonts.googleapis.com/icon?family=Material+Icons';
+```
+
+Run `ng serve` to run your application in develop mode, and navigate to `http://localhost:4200`.
```
Comment (hansl) [inline docs/documentation/stories/include-angular-material.md]: "in development mode".

File: docs/documentation/stories/include-angular-material.md [hunk 10]
```diff
@@ -0,0 +1,152 @@
...
+
+To verify angular material has been set up correctly, change `src/app/app.component.html` to the following...
```
Comment (hansl) [inline docs/documentation/stories/include-angular-material.md]: Capital A and capital M to Angular Material, here and everywhere else.

File: docs/documentation/stories/include-angular-material.md (hunk 4 above)
Comment (jeremymwells) [inline docs/documentation/stories/include-angular-material.md]: That is correct. I considered a separate guide for it, but stitching them together made sense when I considered that they currently ship together for v1.

Comment (jeremymwells) [inline docs/documentation/stories/include-angular-material.md]: @hansl - am I to do a separate PR for the flex stuff?

**Comment by Brocco** [comment]
Can you please rebase
